music_assistant/controllers/music.py

```diff
@@ -842,10 +842,9 @@ async def get_loudness(
             },
         )
         if db_row and db_row["loudness"] != inf and db_row["loudness"] != -inf:
-            loudness = db_row["loudness"]
+            loudness = round(db_row["loudness"], 2)
             loudness_album = db_row["loudness_album"]
-            if loudness_album in (inf, -inf):
-                loudness_album = None
+            loudness_album = None if loudness_album in (inf, -inf) else round(loudness_album, 2)
```

Comment:
```suggestion
            loudness_album = None if loudness_album in (None, inf, -inf) else round(loudness_album, 2)
```
You need to guard this because loudness_album is an optional value

Reply:
Thanks. Updated. I wrongly assumed it could only be `(inf, -inf)` based on the previous code but of course the previous code only re-set it to `None` anyway.